.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,14 +5,12 @@
  
 - [ ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
 - [ ] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
-- [ ] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) beforehand.
-- [ ] Run 
+- [ ] Run the following the [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
```

Comment:
```suggestion
- [ ] Run the following to the [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
```

Reply:
DONE. I also replaced `mvn` with `./mvnw` instead as some users do not have maven installed.